app/code/core/Mage/Catalog/Model/Layer/Filter/Decimal.php

```diff
@@ -184,7 +184,7 @@ public function getRange()
      * Retrieve information about products count in range
      *
      * @param int $range
-     * @return int
+     * @return array
```

Comment:
wrongful (and unprecise) change

Reply:
This one is correct ... see `Mage_Catalog_Model_Resource_Layer_Filter_Decimal::getCount()`